server/app/views/admin/programs/ProgramBlockAllAspectsView.java

```diff
@@ -225,7 +225,8 @@ private DivTag addFormEndpoints(InputTag csrfTag, long programId, long blockId)
   private DivTag blockOrderPanel(Request request, ProgramDefinition program, long focusedBlockId) {
     DivTag ret = div().withClasses("shadow-lg", "pt-6", "w-2/12", "border-r", "border-gray-200");
     ret.with(
-        blockList(request, program, program.getNonRepeatedBlockDefinitions(), focusedBlockId, 0));
+        renderBlockList(
+            request, program, program.getNonRepeatedBlockDefinitions(), focusedBlockId, 0));
```

Comment:
```suggestion
            request, program, program.getNonRepeatedBlockDefinitions(), focusedBlockId, /* level= */ 0));
```

Reply:
not sure if you added or moved this, but ambiguous params should be documented inline.  the `=` syntax will trigger a linter if the param name changes.